lib/CL/devices/almaif/AlmaifDB/AlmaifDBManager.cc

```diff
@@ -0,0 +1,289 @@
+/* AlmaifDBManager.cc - Parses and and responds to queries about AlmaifDB
```

Comment:
```suggestion
/* AlmaifDBManager.cc - Parses and responds to queries about AlmaifDB
```
We need to document the AlmaIFDB somewhere. It would be good to include a paragraph in the docs and not only link to the publication.

Reply:
Documenting the AlmaifDB format would be good at some point. But I don't want do it yet, since there are likely to be more breaking changes, so it would need to be maintained alongside the parser. Added a comment about the separate AFOCL-repo here which contains examples of AlmaIFDB.

Reply:
Yes, but I meant more in general: What it is and why it is?